test/test_bigquery_wrapper.rb

```diff
@@ -25,20 +25,7 @@ def test_configure_json_keyfile
             'dataset'      => 'bigquery_migration_unittest',
             'table'        => 'test',
           }
-          assert_nothing_raised { instance.project }
-          assert_nothing_raised { instance.dataset }
-          assert_nothing_raised { instance.table }
-          assert_nothing_raised { instance.client }
-        end
-
-        def test_configure_json_keyfile_content_json
```

Comment:
Can you fix so that this work https://github.com/medjed/bigquery_migration/blob/ab05ed574ee1a2d7a476630d46de9c3b9b8d9d6f/lib/bigquery_migration/bigquery_wrapper.rb#L732 ?

Reply:
I fixed it, and I understood for `json_keyfile[:content]` works with string of json.
Thanks for your review.